ticdc/ticdc-sink-to-kafka.md

```diff
@@ -258,14 +258,14 @@ Take the following configuration of `dispatchers` as an example:
 [sink]
 dispatchers = [
     {matcher = ['test.*'], partition = "index-value"},
-    {matcher = ['test1.*'], partition = "index-value", index-name = "index1"},
+    {matcher = ['test1.*'], partition = "index-value", index = "index1"},
```

Comment:
Please also update this file: https://github.com/pingcap/docs/blob/e9006b55a24fb806f795d1bdff89ea716b8c14f6/ticdc/ticdc-changefeed-config.md?plain=1#L125